.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:12.18.4
+
+jobs:
+  branch_check:
+    <<: *defaults
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          - v1-dependencies-
+
+      - run: npm install
```

Comment:
shouldn't the CI be using `yarn` for all the things? (we maintain yarn.lock in repo)

Reply:
fixed